.buildkite/test-sample-yamls.yml

```diff
@@ -28,7 +28,7 @@
     - kubectl config set clusters.kind-kind.server https://docker:6443
     # Deploy KubeRay operator
     - pushd ray-operator
-    - IMG=quay.io/kuberay/operator:v1.2.2 make deploy
+    - IMG=quay.io/kuberay/operator:v1.3.0 make deploy
```

Comment:
cc @andrewsykim This is a missing part in our release guide. Would you mind adding this point when you update it to improve the release process? Thanks